packages/fints/src/segments/hicdb.ts

```diff
@@ -51,7 +51,7 @@ export class HICDB extends SegmentClass(HICDBProps) {
             interval: Parse.num(interval),
             orderDay: Parse.num(orderDay),
             lastOrderDate: lastOrder ? Parse.date(lastOrder) : null,
-            creationDate: Parse.date(jsonMessage.GrpHdr.CreDtTm),
+            creationDate: new Date(jsonMessage.GrpHdr.CreDtTm),
```

Comment:
Why did you change this?

Reply:
<img width="769" alt="Screenshot 2020-02-16 at 09 27 46" src="https://user-images.githubusercontent.com/8288065/74602256-b4682380-509e-11ea-8b3d-fd27316b1c9f.png">